run

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+CARGO_INCREMENTAL=1 cargo run --example gltf-animation -- ~/gltf/glTF-Sample-Models/2.0/$1/glTF/$1.gltf
```

Comment:
What's the purpose of this file? I hope it's just helper for you and won't be exposed to the user.

Reply:
It's just for me. I swear I added it to .gitignore...